addons/binding/org.openhab.binding.unifi/src/main/java/org/openhab/binding/unifi/internal/api/UniFiController.java

```diff
@@ -95,7 +97,7 @@ public void login() throws UniFiException {
     public UniFiClient[] getInsights(UniFiSite site) throws UniFiException {
         UniFiControllerRequest<UniFiClient[]> req = newRequest(UniFiClient[].class);
         req.setPath("/api/s/" + site.getName() + "/stat/alluser");
-        req.setQueryParameter("within", 24);
+        req.setQueryParameter("within", 168); // scurb: Changed to 7 days.
```

Comment:
Why did you change it?

Reply:
I changed the insight value in order to increase the chances of finding the client that should be blocked/unblocked. If the clients are not active, the binding queries the insight api endpoint (kind of a history table). If a client has been disconnected for more than a day (previous value) then I thought a week should be more fair. I don’t see any negative perspective in increasing the value more then maybe the ”load” on the controller/oh binding in generating or parsing the result.

Reply:
Yes, exactly. The client gets disconnected which makes it only visible when doing Insights-lookup. And for clients that has been disconnected I wanted to make it possible to find them even though a couple of days has passed (one week).